From b08275d549bb12a09a420ef9ccaeb0974e00008a Mon Sep 17 00:00:00 2001
From: Stephen Kitt <steve@sk2.org>
Date: Sat, 6 Feb 2021 22:57:52 +0100
Subject: [PATCH] Stop using a header for wixl-heat

Having wixl and wixl-heat use the same header means that, if wixl-heat
is built after wixl, anything relying on the full wixl.h subsequently
will fail. wixl-heat doesn't need a header, so we can stop using it.

Fixes: #31
Signed-off-by: Stephen Kitt <steve@sk2.org>
---
 tools/wixl/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/wixl/meson.build b/tools/wixl/meson.build
index bcce524..8a056fa 100644
--- a/tools/wixl/meson.build
+++ b/tools/wixl/meson.build
@@ -47,8 +47,6 @@ wixl_heat = executable('wixl-heat',
   'wixl-heat.vala',
   config_h,
   vala_args: [
-    '-H', 'wixl.h',
-    '--use-header',
     '--enable-experimental',
     '--pkg', 'config',
     '--pkg', 'posix',
-- 
GitLab

